src/modules/core/components/checkbox/checkbox.module.scss

```diff
@@ -0,0 +1,118 @@
+@use '@/styles/variables.module';
+
+.container {
+  display: block;
+  position: relative;
+  cursor: pointer;
+  font-size: 22px;
+  user-select: none;
+  padding: 9px;
```

Comment:
use `spacing` util that based on 8px sizes.

```
@use '@/styles/utils';

padding: utils.spacing(1);
```

utils.spacing(1) is the same as 8px; utils.spacing(2) is the same as 16px and so on...